components/layout/DocsLayout.js

```diff
@@ -13,6 +14,13 @@ import ArrowRight from '../icons/ArrowRight'
 import AnnouncementRemainingDays from '../campaigns/AnnouncementRamainingDays'
 import AnnouncementHero from '../campaigns/AnnoucementHero'
 
+function generateEditLink(post) {
+  if (post.slug.includes('/specifications/')) {
+    return <a target="_blank" rel="noopener noreferrer" href={`https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md`} className="ml-1 underline">Edit this page on Github</a>
```

Comment:
@derberg @magicmatatjahu What should we link to for old(/previous) specs? Leave it as it is or Remove this for old specs?(This will only work if we know what the latest spec is).

Reply:
for old and latest always link to master
for release candidate link to release candidate branch. Did we figure already how to cleanup release candidates after release?

Reply:
> for old and latest always link to master
for release candidate link to release candidate branch.

Okay, cool.

> Did we figure already how to cleanup release candidates after release?

Not at the moment. I'm gonna work on this right after this. 

Just for clarification, pre-release should be named as `2021-06-pre-release.md`, right?
and for release, it should be `2021-06.md`?

Reply:
pre-release is `v2.1.0-2021-06-release.md` and release is `v2.1.0.md`

`v2.1.0-2021-06-release.md` = `v` + semver version + year + month + `release` sufix

`2.0.0.md` should be `v2.0.0.md`